test/edu-benefits/1990-rjsf/config/applicantInformation.unit.spec.jsx

```diff
@@ -1,10 +1,9 @@
 import React from 'react';
 import { expect } from 'chai';
 import ReactTestUtils from 'react-dom/test-utils';
-import { getFormDOM } from '../../../util/schemaform-utils';
 import moment from 'moment';
 
-import { DefinitionTester } from '../../../util/schemaform-utils.jsx';
+import { DefinitionTester, getFormDOM } from '../../../util/schemaform-utils.jsx';
```

Comment:
Nope, but I think we'll end up turning on the rules to prevent using extensions and I can auto-fix them then.